utils_cv/detection/model.py

```diff
@@ -353,14 +461,14 @@ def predict_dl(
             threshold: iou threshold for a positive detection. Note: set
             threshold to None to omit a threshold
 
-        Returns a list of DetectionBbox
+        Returns a list of results
         """
         pred_generator = self.predict_batch(dl, threshold=threshold)
-        det_bboxes = [pred for preds in pred_generator for pred in preds]
-        return det_bboxes
+        res = [pred for preds in pred_generator for pred in preds]
```

Comment:
Can simplify to return [pred for preds in pred_generator for pred in preds]

Reply:
Done